Source/WebCore/platform/audio/cocoa/AudioSampleDataConverter.h

```diff
@@ -66,6 +66,7 @@ class AudioSampleDataConverter {
         AudioConverterRef m_audioConverter { nullptr };
     };
 
+    bool m_enabled { true };
```

Comment:
`m_enabled` sounds like it controls the entire class. Maybe something like `m_latencyAdaptationEnabled`?

Reply:
Sounds good, done